2022/templates/cube.svg

```diff
@@ -0,0 +1,1309 @@
+         x="8623.875"
+         y="8969.5322"
+         id="text1011"><tspan
+           sodipodi:role="line"
+           id="tspan1009"
+           x="8623.875"
+           y="8969.5322"
+           style="stroke-width:0.264583">Arena</tspan></text>
+      <path
+         style="fill:none;stroke:#000000;stroke-width:56.5962;stroke-linecap:butt;stroke-linejoin:round;stroke-miterlimit:4;stroke-dasharray:452.769, 113.192, 56.5962, 113.192;stroke-dashoffset:0;stroke-opacity:1"
+         d="M 999.99993,12694.961 H 6124.9996"
+         id="path1046"
+         inkscape:connector-curvature="0" />
+      <path
+         style="fill:none;stroke:#000000;stroke-width:64.952;stroke-linecap:butt;stroke-linejoin:round;stroke-miterlimit:4;stroke-dasharray:519.615, 129.903, 64.952, 129.903;stroke-dashoffset:0;stroke-opacity:1"
+         d="M 6175.0006,12690.783 H 12925"
+         id="path1046-3"
+         inkscape:connector-curvature="0" />
+      <path
+         style="fill:none;stroke:#000000;stroke-width:56.5962;stroke-linecap:butt;stroke-linejoin:round;stroke-miterlimit:4;stroke-dasharray:452.769, 113.192, 56.5962, 113.192;stroke-dashoffset:0;stroke-opacity:1"
+         d="M 12925.004,12718.26 V 5018.2604"
+         id="path1046-6"
+         inkscape:connector-curvature="0" />
+      <path
+         transform="translate(-1.1743019e-5)"
+         style="fill:none;stroke:#000000;stroke-width:50;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:400, 100, 50, 100;stroke-dashoffset:0;stroke-opacity:1"
+         d="M 7399.9997,2799.9996 V 4999.9994"
+         id="path1109"
+         inkscape:connector-curvature="0" />
+      <text
+         xml:space="preserve"
+         style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:264.583px;line-height:1.25;font-family:sans-serif;-inkscape-font-specification:'sans-serif, Normal';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-feature-settings:normal;text-align:start;letter-spacing:0px;word-spacing:0px;writing-mode:lr-tb;text-anchor:start;display:inline;fill:#000000;fill-opacity:1;stroke:none;stroke-width:0.264583"
+         x="2124.3699"
+         y="8364.0566"
+         id="text1052"><tspan
+           sodipodi:role="line"
+           id="tspan1050"
+           x="2124.3699"
+           y="8364.0566"
+           style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:264.583px;font-family:sans-serif;-inkscape-font-specification:'sans-serif, Normal';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-feature-settings:normal;text-align:start;writing-mode:lr-tb;text-anchor:start;stroke-width:0.264583">Staging Area</tspan></text>
```

Comment:
Is it only me, or is the staging area here pretty large, perhaps more than it needs to be?

Reply:
Staging just occupies the space between the arena and the wall. Since it's significantly more awkward to light the arena evenly if it's off-center there isn't a lot we can do about staging.